### DISCUSSION TITLE
Question: UWP - Could not draw in InkCanvas once placed inside a ContentDialog

### DISCUSSION THREAD

**Discussion by sudathk** [microsoft/microsoft-ui-xaml#7477] [Q&A]
Upon Updating Microsoft.UI.Xaml NuGet package from version 2.2.190830001 to 2.5.0 unable to draw in InkCanvas once placed inside a ContentDialog. 
That functionality is not possible all versions released after 2.2.190830001.
Please advise.

**Comment by StephenLPeters** [comment]
@sudathk Can you give more details on what behavior you're seeing? Do you have a repro app on hand that you could share?

**Comment by sudathk** [comment]
@StephenLPeters 

Sample application has been attached for you to have a look at.

Steps to reproduce the issue.
Run the app.
Step1:
Hit the Show Dialog to open the dialog box and try drawing inside the Canvas.
- Result: does not draw
<Application.Resources>
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
</Application.Resources>

Step2: Remove the above XMAL style include and follow the instructions in Step1
- Result: It draws

[ContentDialogCanvas.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6630681/ContentDialogCanvas.zip)

Hope above steps help to discover the issue.
